src/pages/LogInWithShortLivedAuthTokenPage.js

```diff
@@ -35,12 +37,23 @@ const propTypes = {
 
     ...withLocalizePropTypes,
 
-    /** Whether the short lived auth token is valid */
-    isTokenValid: PropTypes.bool,
+    /** The details about the account that the user is signing in with */
+    account: PropTypes.shape({
+        /** Whether a sign is loading */
+        isLoading: PropTypes.bool,
+    }),
+
+    /** Props to detect online status */
+    network: networkPropTypes,
```

Comment:
It looks like everywhere else in the app it uses `.isRequired` for this prop. Let's do that here.